draft-ietf-rats-corim.md

```diff
@@ -1259,6 +1259,22 @@ for appraisal until a CoBOM which activates that tag has been received and
 successfully processed. All the tags listed in the CoBOM must be activated in
 the same transaction, i.e., either all or none.
 
+The number of CoBOMs required in a given supply chain eco-system is dependent on
+Verifier Owner's appraisal policy, which is often driven by the complexity and nature of
+use case.
+
+In a certain use case CoBOM's may not be required at all. For example, where only limited CoRIMs
+are produced by a single authority. In such cases any tags in a CoRIM received by a Verifier
+are activated immediately and treated valid for appraisal.
+
+However, a more common case is the one where Verifier receives multiple CoRIMs containing
+Reference Values and Endorsements from multiple Reference Value providers and Endorsers.
+In such cases an Integrator or a Supplier authority is often designated to issue a single CoBOM
+to activate all the tags supplied in the CoRIMs.
```

Comment:
```suggestion
Alternatively, a Verifier Owner may choose to delegate approval of CoRIM tags to a CoBOM issuer.
The Verifier would be configured so that it only activates CoRIM tags which have been explicitly
approved by a CoBOM issued by an appropriate authority.
```

Reply:
Again the focus is on describing the use case situation rather than the result which drives the Policy, hence retained and tidied the existing text.